dist-files/Makefile.staticlibrary

```diff
@@ -0,0 +1,50 @@
+#
+#  Example of how to build and install locally as a static library
+#
+#  Usage:
+#
+#    $ make -f Makefile.staticlibrary
+#    $ sudo make -f Makefile.staticlibrary install
+#    $ make -f Makefile.staticlibrary duk  # --> example 'duk' linked to static libduktape
+#
+#    $ ls -l duk
+#    -rwxrwxr-x 1 duktape duktape 413408 Nov 30 15:48 duk
+#
+#  Based on: http://tldp.org/HOWTO/Program-Library-HOWTO/static-libraries.html
+
+# Change to actual path for actual distribution packaging.
+INSTALL_PREFIX = /usr/local
+
+# The 'noline' variant may be more appropriate for some distributions; it
+# doesn't have #line directives in the combined source.
+DUKTAPE_SRCDIR = ./src
+#DUKTAPE_SRCDIR = ./src-noline
+
+AR = ar
+CC = gcc
```

Comment:
I found myself wanting to build a static version of duktape. (using this pull-request and nix: `nix build nixpkgs#pkgsStatic.duktape`, with the patch applied) and those two lines should be:
````
AR ?= ar
AR := $(AR)
CC ?= gcc
CC := $(CC)
``
just like in the Makefile.sharedlibrary on the master branch. So that it uses my x86_64-unknown-linux-musl-gcc from $CC of the Environment instead of just gcc.

Reply:
I updated the PR but it is opened since 2021 ...